runners/direct-java/src/main/java/org/apache/beam/runners/direct/WatermarkManager.java

```diff
@@ -965,15 +967,19 @@ private void applyNUpdates(int numUpdates) {
   }
 
   private void applyPendingUpdate(PendingWatermarkUpdate<ExecutableT> pending) {
-    CommittedResult<ExecutableT> result = pending.getResult();
-    ExecutableT transform = result.getExecutable();
+    ExecutableT transform = pending.getExecutable();
```

Comment:
Are transform and executable are interchangeable term? Or, is transform a subset of executable?